### PR TITLE
change id to userinfo for more info, edited message commands.

### PR DESCRIPTION
**Pull request by AutumnClove**
Please make sure these boxes are checked (put an 'x' inside them) **before** creating the pull request.

- [x] I have tested my changes against the `review` branch (the latest developmental version), and this pull request is targeting that branch as a base
- [x] I have tested my changes on Python 3.5/3.6

----

### Description
ID is so boring why not provide some better info

### CODE REVIEW

**Comment by jayktaylor** [comment]
Please check why this failed a CI build and fix the issues.

**Comment by AutumnClove** [comment]
@jaydenkieran 
Please help idk how to fix the indentation, githubs online editor confuses me inb4 use external, I did I copy pasted it over.

**Comment by jayktaylor** [comment]
You should use something like Visual Studio Code and update your repo the proper way.

**Comment by AutumnClove** [comment]
I'm assuming the proper way is using github desktop? Just tried that and while the indentation is correct on my side, upon uploading it gets messed up =.=

**Comment by MattBSG** [comment]
Just to add to this, the `id` command is currently depreciated.

File: musicbot/bot.py [hunk 1]
```diff
@@ -1317,18 +1317,33 @@ def _gen_embed(self):
...
+        status = member.status
+        game = member.game
+        server = member.server
+        nickname = member.nick
+        color = member.colour
+        top_role = member.top_role
+        creation_date = member.created_at
+        avatar = member.avatar_url if member.avatar_url else member.default_avatar_url
+        user_id = member.id
+        return Response(self.str.get('cmd-user', """
```
Comment (MattBSG) [inline musicbot/bot.py]: Please commit the updated `en.json` i18n file with the added string this references

**Review by MattBSG — CHANGES_REQUESTED** [review]

**Review by MattBSG — APPROVED** [review]

**Review by obi23lipnik — APPROVED** [review]

**Review by ImthePeach — COMMENTED** [review]
This should have a embed availability as well for cleaner look, user info never looks good when it's not in an embed. Command should be called `info` or `userinfo` instead of `user` for sanity reasons. You aren't asking for what user is this, you are asking for info on the user.

File: musicbot/bot.py [hunk 2]
```diff
@@ -1317,18 +1318,66 @@ def _gen_embed(self):
...
+            {command_prefix}userinfo [@user]
 
-        Tells the user their id or the id of another user.
+        Provides some info on a user, if no-one mentioned returns info on message author.
         """
         if not user_mentions:
-            return Response(self.str.get('cmd-id-self', 'Your ID is `{0}`').format(author.id), reply=True, delete_after=35)
+            stripped = len(self.config.command_prefix)+9
+            int(stripped)
+            ph = message.content[stripped:]
```
Comment (AutumnClove) [inline musicbot/bot.py:1331]: ph stands for placeholder, not pornhub LUL

**Comment by AutumnClove** [comment]
Holy crud it's almost been a year?

**Pull request closed without merging**